device-types/Palo Alto/PA-5450.yaml

```diff
@@ -0,0 +1,44 @@
+---
+manufacturer: Palo Alto
+model: PA-5450
+slug: palo-alto-pa-5450
+part_number: PA-5450
+u_height: 5.0
+is_full_depth: true
+airflow: front-to-rear
+console-ports:
+  - name: console port (Micro USB)
+    type: usb-micro-a
+    label: CONSOLE
+  - name: console port (RJ-45)
```

Comment:
There are two ports, one Micro USB type A and one RJ45.  We can only use one at a time.  There is no reference to a console interface in the running configuration.  The best we can do is to use the external printed label.  I am also moving this to the management card since the console port is not on the chassis.